### PR TITLE
Item exercise

### PR DESCRIPTION
**Pull request by JuanButronC**
Just doubted about implement or not a uuid getter method, 'cause is not mentioned.
I Hope the exercise is ok. Ty.

### CODE REVIEW

File: Ejercicio2/Item.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package Ejercicio2;
+
+public interface Item {
```
Comment (ichi0915) [inline Ejercicio2/Item.java:3]: add toString() method

File: Ejercicio2/Weapon.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    public Integer amount;
+    public Integer amountLimit;
+    public String type;
+    public Double power;
+
+    public abstract String getType();
+
+    public abstract void setType(String type);
+
+    public abstract Double getPower();
```
Comment (ichi0915) [inline Ejercicio2/Weapon.java]: This Method Shouldn't be abstract, make it a concrete method

File: Ejercicio2/Weapon.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    public abstract void setPower(Double power);
```
Comment (ichi0915) [inline Ejercicio2/Weapon.java]: This Method Shouldn't be abstract, make it a concrete method

**Comment by ichi0915** [comment]
yeah the uuid getter method is not necessary, and everything is good just that I changed the UML last minute and now you need to add the toString and change the setter and getter of Type to abstract.

**Comment by JuanButronC** [comment]
Hi, Ichi, I've already did the changes, I think there are in this same pull request but with a different commit called "Added toString method..." because I didn't be able to create another pull request. Thanks!

**Comment by ichi0915** [comment]
Awesome, and yes when you make changes to your branch git automatically updates the PR

**Pull request merged**